dali/test/python/test_dali_tf_dataset.py

```diff
@@ -170,96 +45,21 @@ def test_different_num_shapes_dtypes():
 
 
-def _test_tf_dataset_multigpu():
-    num_devices = num_available_gpus()
-    batch_size = 8
-    num_threads = 4
-
-    shapes = (
-        (batch_size, 3, 224, 224),
-        (batch_size, 1, 1),
-        (batch_size, 1))
-    dtypes = (
-        tf.float32,
-        tf.int32,
-        tf.int16)
-
-    dataset_results = []
-    initializers = [tf.compat.v1.global_variables_initializer()]
-    ops_to_run = []
-
-    for device_id in range(num_devices):
-        with tf.device('/gpu:{0}'.format(device_id)):
-            dataset_pipeline = TestPipeline(batch_size, num_threads, 'gpu', device_id, device_id, num_devices)
-            daliset = dali_tf.DALIDataset(
-                pipeline=dataset_pipeline,
-                batch_size=batch_size,
-                output_shapes=shapes,
-                output_dtypes=dtypes,
-                num_threads=num_threads,
-                device_id=device_id)
-
-            iterator = tf.compat.v1.data.make_initializable_iterator(daliset)
-            initializers.append(iterator.initializer)
-
-            ops_to_run.append(iterator.get_next())
-
-
-    standalone_pipeline = TestPipeline(
-        batch_size, num_threads, device = 'gpu', device_id = 0)
-    standalone_pipeline.build()
-
-    dataset_size = standalone_pipeline.epoch_size("Reader")
-    iterations = dataset_size // batch_size
-
-    with tf.compat.v1.Session() as sess:
-        sess.run(initializers)
-        for _ in range(iterations):
-            dataset_results.append(sess.run(ops_to_run))
-
-    standalone_results = []
-    for _ in range(iterations):
-        standalone_results.append(
-            tuple(result.as_cpu().as_array() for result in standalone_pipeline.run()))
-
-    assert len(dataset_results) == iterations
-    for it in range(iterations):
-        assert len(dataset_results[it]) == num_devices
-        for device_id in range(num_devices):
-            batch_id = iterations - ((it + device_id * (iterations // num_devices)) % iterations) - 1
-            it_id = iterations - it - 1
-            assert np.array_equal(
-                standalone_results[it_id][0],
-                dataset_results[batch_id][device_id][0])
-            assert np.array_equal(
-                standalone_results[it_id][1],
-                dataset_results[batch_id][device_id][1])
-            assert np.array_equal(
-                standalone_results[it_id][2],
-                dataset_results[batch_id][device_id][2])
-
-
-class PythonOperatorPipeline(Pipeline):
-    def __init__(self):
-        super(PythonOperatorPipeline, self).__init__(1, 1, 0, 0)
-        self.python_op = ops.PythonFunction(function=lambda: np.zeros((3, 3, 3)))
-
-    def define_graph(self):
-        return self.python_op()
-
-
 @raises(RuntimeError)
 def test_python_operator_error():
-    dataset_pipeline = PythonOperatorPipeline()
+    pipeline = Pipeline(1, 1, 0, 0)
+    with pipeline:
+        output = fn.python_function(function=lambda: np.zeros((3, 3, 3)))
```

Comment:
it's not clear to me what is the expected error. Maybe the name of the test case should say it

Reply:
`PythonFunction` is not allowed with TF for now. Changed the name of the test to include that inforamation.